diff --git a/node_modules/playwright-core/lib/coreBundle.js b/node_modules/playwright-core/lib/coreBundle.js
index 8c08494..01835a9 100644
--- a/node_modules/playwright-core/lib/coreBundle.js
+++ b/node_modules/playwright-core/lib/coreBundle.js
@@ -58827,7 +58827,7 @@ var init_response = __esm({
           addSection("Result", this._results);
         if (this._context.config.codegen !== "none" && this._code.length)
           addSection("Ran Playwright code", this._code, "js");
-        const tabSnapshot = this._context.currentTab() ? await this._context.currentTabOrDie().captureSnapshot(this._includeSnapshotRoot, this._includeSnapshotDepth, this._includeSnapshotBoxes, this._clientWorkspace) : void 0;
+        const tabSnapshot = (this._includeSnapshot !== "none" && this._context.currentTab()) ? await this._context.currentTabOrDie().captureSnapshot(this._includeSnapshotRoot, this._includeSnapshotDepth, this._includeSnapshotBoxes, this._clientWorkspace) : void 0;
         const tabHeaders = await Promise.all(this._context.tabs().map((tab2) => tab2.headerSnapshot()));
         if (this._includeSnapshot !== "none" || tabHeaders.some((header) => header.changed)) {
           if (tabHeaders.length !== 1)
